packages/block-library/src/navigation/index.php

```diff
@@ -121,6 +121,19 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 
 	unset( $attributes['rgbTextColor'], $attributes['rgbBackgroundColor'] );
 
+	$script_path                 = __DIR__ . '/navigation/frontend.js';
+	$should_load_frontend_script = $attributes['isResponsive'] && ! wp_script_is( 'core_block_navigation_load_frontend_scripts' ) && file_exists( $script_path );
```

Comment:
Is there a possibility that the script wouldn't exist (`file_exists( $script_path )`)?

Reply:
Only if it is manually removed, so here we're just being perhaps unnecessarily careful. I'll skip that check for clarity's sake, and because I think an error would be warranted if files like these were actually manually deleted.